sdk/communication/Azure.Communication.Chat/tests/Infrastructure/ChatLiveTestBase.cs

```diff
@@ -14,7 +15,7 @@ public class ChatLiveTestBase : RecordedTestBase<ChatTestEnvironment>
     {
         public ChatLiveTestBase(bool isAsync) : base(isAsync)
         {
-            Sanitizer = new ChatRecordedTestSanitizer();
+            Sanitizer = new CommunicationRecordedTestSanitizer();
```

Comment:
nit: We can convert this to expression body constructor to make it look more like other parts of our code

Reply:
done